apps/mobile/e2e/run-cross-platform-e2e.sh

```
@@ -0,0 +1,105 @@
+# Prerequisites:
+#   - iOS Simulator with Drafto app installed and user logged in
+#   - Android Emulator with Drafto app installed and user logged in
+#   - Maestro CLI installed
+#   - .env.local sourced (E2E_TEST_EMAIL, E2E_TEST_PASSWORD, SUPABASE vars)
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+REPO_ROOT="$(cd "$SCRIPT_DIR/../../.." && pwd)"
+WEB_DIR="$REPO_ROOT/apps/web"
+MOBILE_DIR="$REPO_ROOT/apps/mobile"
+MAESTRO="$HOME/.maestro/bin/maestro"
+ADB="/Users/jakub/Library/Android/sdk/platform-tools/adb"
```

Shellcheck (0.11.0)

[warning] ADB appears unused. Verify use (or export if used externally).

(SC2034)